src/assets/scss/index.scss

```diff
@@ -125,9 +135,10 @@ body.dark {
   --button-background-secondary: transparent;
   --button-text-secondary: #2dbae6;
   --sidebar-text: var(--primary);
-  --content-widget-text: #dadbe6;
+  --content-widget-text: #AFB3C0;

```

Comment:
Optional: It seems the file is inconsistent about letters in color values (some are all lowercases, but others are uppercase, too). If you have capacity, check whether this can be refactored so that all colors use the same case.